index.html

```diff
@@ -222,12 +220,36 @@ <h3>Ruby</h3>
                     </li>
                 </ul>
             </div>
-           
+
             <button type="button">See Project </button>
-           
+
         </div>
 
+
     </section>
+    <footer class="f-contact">
+        <h3>CONTACT US</h3>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that
+            needs
+            coding. I’d love to help with it</p>
+        
+        <form action="https://formspree.io/f/moqrdewo" method="post">
+            <input type="text" name ="full name" placeholder="Text input" maxlength="30" required>
+            <input type="email" name="email" placeholder="khanetmoi@gmail.com" required>
+            <textarea cols="30" rows="10" name="message" placeholder="Enter text here"></textarea>
```

Comment:
[REQUIRED] Please kindly make sure that the `message input` only accepts up to 500 characters and it is a `required` field meaning that the user can not leave this `message field `blank, as it is required by this project.

_hint: it can be done with `maxlength` and `required` attribute_

Using built-in ` HTML5 client-side form validation` will ensure that all required form controls are filled out, in the correct format, making our lives at the `back-end` much easier.